dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/parameters/PredictorJobParameters.java

```diff
@@ -47,12 +50,20 @@
  */
 @Getter
 @Setter
+@Builder
 @NoArgsConstructor
+@AllArgsConstructor
 public class PredictorJobParameters implements JobParameters
 {
+    /**
+     * Today plus n days
```

Comment:
Thanks for adding the comments! I suggest making this and the following comment:
```
    /**
     * Today plus n days (n can be negative)
     */
```
If n is positive it will be a future date and if n is negative it will be a past date. Both positive and negative offsets have valid use cases for both predictor start and predictor end. "Today plus n days" is technically correct. I just want the reader to know that n can be a negative number. I would not want to say "Today plus or minus n days" because that might give the false impression that a positive n selects a date range of both plus and minus from today. So that's why I suggest adding "(n can be negative)". If you like, "(can be negative)" also works, or anything else that informs the reader that n may be negative.